depends/README.md

```diff
@@ -100,10 +100,19 @@ options will be passed to bitcoin's configure. In this case, `--disable-wallet`.
 
 ### Additional targets
 
-    download: run 'make download' to fetch all sources without building them
-    download-osx: run 'make download-osx' to fetch all sources needed for macOS builds
-    download-win: run 'make download-win' to fetch all sources needed for win builds
-    download-linux: run 'make download-linux' to fetch all sources needed for linux builds
+    make download # fetch all sources without building them
+
+For macOS:
+
+    make download-osx # fetch all sources needed for macOS builds
+
+For Windows:
+
+    make download-win # fetch all sources needed for win builds
+
+For Linux:
+
+    make download-linux # to fetch all sources needed for linux builds
```

Comment:
It makes it more confusing, IMO.
What does "For macOS" mean: a host system or a target one?

Let's leave it as is :)

Reply:
I agree with @hebasto that this is less clear than the way it is currently.